cmd/delete.go

```diff
@@ -152,9 +168,37 @@ func (c deleteConfig) Prompt() (deleteConfig, error) {
 
 	dc := deleteConfig{}
 
-	return dc, survey.AskOne(
-		&survey.Input{
-			Message: "Function to remove:",
-			Default: deriveName(c.Name, c.Path)},
-		&dc.Name, survey.WithValidator(survey.Required))
+	var qs = []*survey.Question{
+		{
+			Name: "name",
+			Prompt: &survey.Input{
+				Message: "Function to remove:",
+				Default: deriveName(c.Name, c.Path)},
+			Validate: survey.Required,
+		},
+		{
+			Name: "all",
+			Prompt: &survey.Confirm{
+				Message: "Do you want to delete all resources?",
+				Default: true,
+			},
+		},
+	}
+	answers := struct {
+		Name string
+		All  bool
+	}{}
+
+	err := survey.Ask(qs, &answers)
+	if err != nil {
+		return dc, err
+	}
+
+	dc.Name = answers.Name
+	dc.DeleteAll = answers.All
+	dc.Namespace = c.Namespace
+	dc.Path = c.Path
+	dc.Verbose = c.Verbose
```

Comment:
I am pretty sure this mapping to a _new_ config object is not necessary and we could just use c directoy, if the original config `c` is treated as defaults (defaults which may have been mutated by command flags before they were parsed in its constructor).